python/core/composer/qgslayoutmanager.sip

```diff
@@ -0,0 +1,34 @@
+
+class QgsLayoutManager : QObject
+{
+%TypeHeaderCode
+#include <qgslayoutmanager.h>
+%End
+  public:
+
+    explicit QgsLayoutManager( QgsProject* project = 0 );
+
+    ~QgsLayoutManager();
+    bool addComposition( QgsComposition* composition /Transfer/ );
+    bool removeComposition( QgsComposition* composition );
+
+    void clear();
+    QList< QgsComposition * > compositions() const;
+    QgsComposition *compositionByName( const QString &name ) const;
+    bool readXml( const QDomElement &element, const QDomDocument &doc );
+    QDomElement writeXml( QDomDocument &doc ) const;
+    bool saveAsTemplate( const QString &name, QDomDocument &doc ) const;
+    QgsComposition *duplicateComposition( const QString &name, const QString &newName );
```

Comment:
No, it's not required - the new composition is automatically added to the manager and ownership remains with the manager.